Softeq.XToolkit.Remote.Tests/Client/HttpMessageHandlerProviderTests.cs

```diff
@@ -11,10 +11,36 @@ public class HttpMessageHandlerProviderTests
     {
         [Fact]
         public void CreateDefaultHandler_Always_ReturnsHttpMessageHandler()
+        {
+            var result = HttpMessageHandlerProvider.CreateOrGetCachedDefaultHandler();
+
+            Assert.IsAssignableFrom<HttpMessageHandler>(result);
+        }
+
+        [Fact]
+        public void CreateOrGetCachedDefaultHandler_Always_ReturnsHttpMessageHandler()
```

Comment:
This test (and the one above) actually don't test anything (or it tests that C# is a strongly typed language 🙂) - we already know that the result is of type `HttpMessageHandler` because it is the return type of the method.
The idea might be to check if result is not null - that might have more cense